src/index.html

```diff
@@ -5,9 +5,349 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="images/favicon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#" class="header__logo logo header__logo--link">
+          <img
+            src="images/logo.png"
+            alt="Miami Condo Kings logo"
+            class="header__logo logo header__logo--img"
+          >
+        </a>
+        <input
+          type="checkbox"
+          class="header__toggler header__toggler--input"
+          id="toggle"
+        >
+        <label for="toggle" class="header__toggler header__toggler--label"></label>
+        <a href="tel: +1 234 555-55-55" class="header__link header__link--icon"></a>
+        <a href="tel: +1 234 555-55-55" class="header__link header__link--telephone">+1 234 555-55-55</a>
```

Comment:
переноси большие атрибуты 
```suggestion
        <a
          href="tel: +1 234 555-55-55"
          class="header__link header__link--telephone"
        >
          +1 234 555-55-55
        </a>
```